functions/Add-DbaAgDatabase.ps1

```diff
@@ -185,10 +185,10 @@ function Add-DbaAgDatabase {
                     $secondaryInstanceReplicaName = $secondaryInstanceReplicaName, $secondaryInstance.InstanceName -join "\"
                 }
 
-                $agreplica = Get-DbaAgReplica -SqlInstance $Primary -SqlCredential $SecondarySqlCredential -AvailabilityGroup $ag.name -Replica $secondaryInstanceReplicaName
+                $agreplica = Get-DbaAgReplica -SqlInstance $Primary -SqlCredential $SqlCredential -AvailabilityGroup $ag.name -Replica $secondaryInstanceReplicaName
```

Comment:
The reason sqlcredential is used is because it connects to the primary to get the info about the secondaries. the reason it was not returning anything before was that  `-Replica $secondaryInstanceReplicaName` was using an incomplete name.